lua/simfphys_darkrp_modules/properties/client/cl_main.lua

```diff
@@ -0,0 +1,10 @@
+hook.Add( "OnEntityCreated", "CFC_OnSimfphysCreated", function( ent )
+    if ent:GetClass() ~= "gmod_sent_vehicle_fphysics_base" then return end
```

Comment:
```suggestion
    if not IsValid( ent ) then return end
    if not simfphys.IsCar( ent ) then return end
```
maybe that would lock it down some more